testing/src/test/java/org/oppia/android/testing/robolectric/OppiaShadowTrafficStatsTest.kt

```diff
@@ -31,6 +32,14 @@ private val oppiaShadowTrafficStats by lazy { OppiaShadowTrafficStats() }
   shadows = [OppiaShadowTrafficStats::class]
 )
 class OppiaShadowTrafficStatsTest {
+
+  @Inject
+  lateinit var context: Context
+
+  private val oppiaShadowTrafficStats by lazy {
+    Shadow.extract(TrafficStats()) as OppiaShadowTrafficStats
```

Comment:
Ditto here, though we may need to use a function, instead, since ``TrafficStats`` probably needs to be passed in (do we really want a singleton, or is this meant to be for static state?).

Reply:
done. 
replaced the instance call to a more reliable Shadow.newInstanceOf() -- does that work? 

And in case of the state, we'd need a static state since ShadowTrafficStats is static and we're basically just replacing it in robolectric.